packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -123,6 +123,7 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
     let { selectedDate, navigatedDate, isMonthPickerVisible, isDayPickerVisible } = this.state;
     let onHeaderSelect = showMonthPickerAsOverlay ? this._onHeaderSelect : undefined;
     let monthPickerOnly = !showMonthPickerAsOverlay && !isDayPickerVisible;
+    let overlayedWithButton = showMonthPickerAsOverlay && showGoToToday;
```

Comment:
> let overlayedWithButton = showMonthPickerAsOverlay && showGoToToday; [](start = 3, length = 69)

Why does the style have to be different for this specific case? What if we wanted to not have month picker overlay but wanted the today button? We would still want the style to look good but it looks like we would still get the old behavior in that case?

Is this something that can be fixed in the date picker by setting the size of the container properly?

Reply:
Per comments offline, let's check this in to fix the immediate problem, would be good to come up with a better solution to avoid these one-off checks

---
In reply to: [157331043](https://github.com/OfficeDev/office-ui-fabric-react/pull/3531#discussion_r157331043) [](ancestors = 157331043)